traffic_ops/traffic_ops_golang/server/servers.go

```diff
@@ -767,17 +774,18 @@ func getServers(h http.Header, params map[string]string, tx *sqlx.Tx, user *auth
 	// Query Parameters to Database Query column mappings
 	// see the fields mapped in the SQL query
 	queryParamsToSQLCols := map[string]dbhelpers.WhereColumnInfo{
-		"cachegroup":       dbhelpers.WhereColumnInfo{"s.cachegroup", api.IsInt},
-		"parentCachegroup": dbhelpers.WhereColumnInfo{"cg.parent_cachegroup_id", api.IsInt},
-		"cdn":              dbhelpers.WhereColumnInfo{"s.cdn_id", api.IsInt},
-		"id":               dbhelpers.WhereColumnInfo{"s.id", api.IsInt},
-		"hostName":         dbhelpers.WhereColumnInfo{"s.host_name", nil},
-		"physLocation":     dbhelpers.WhereColumnInfo{"s.phys_location", api.IsInt},
-		"profileId":        dbhelpers.WhereColumnInfo{"s.profile", api.IsInt},
-		"status":           dbhelpers.WhereColumnInfo{"st.name", nil},
-		"topology":         dbhelpers.WhereColumnInfo{"tc.topology", nil},
-		"type":             dbhelpers.WhereColumnInfo{"t.name", nil},
-		"dsId":             dbhelpers.WhereColumnInfo{"dss.deliveryservice", nil},
+		"cachegroup":       {"s.cachegroup", api.IsInt},
+		"parentCachegroup": {"cg.parent_cachegroup_id", api.IsInt},
+		"cdn":              {"s.cdn_id", api.IsInt},
+		"id":               {"s.id", api.IsInt},
+		"hostName":         {"s.host_name", nil},
+		"physLocation":     {"s.phys_location", api.IsInt},
+		"profileId":        {"s.profile", api.IsInt},
+		"status":           {"st.name", nil},
+		"topology":         {"tc.topology", nil},
+		"type":             {"t.name", nil},
+		"dsId":             {"dss.deliveryservice", nil},
+		"orgType":          {":orgType", nil},
```

Comment:
Since `orgType` isn't a supported query param, I think we should probably just
```
t.name != `+tc.OriginTypeName+`
```
in the query above and leave it out of this map and remove L821.

Reply:
Changed in 1076fef558